lib/inspector/inspector_tree_canvas.dart

```diff
@@ -0,0 +1,394 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+library inspector_tree_canvas;
```

Comment:
I'm curious if we need `library` directives? I normally only add them if I need someplace to hang dartdoc comments off of.